README.md

```diff
@@ -50,6 +50,29 @@ To get the config & update eve image stored in dist/amd64/installer/rootfs.img
  eden controller -m adam:// edge-node eveimage-update dist/amd64/installer/rootfs.img
 ``` 
 
+## Connect to eve console
+
+``` 
+eden eve console
+``` 
+
+## Run applications on EVE
+
+Deploy nginx server from dockerhub. Expose port 80 of the container to port 8081 of eve. 
+``` 
+eden pod deploy -p 8081:80 docker://nginx
+``` 
+
+Deploy a VM from Openstack. Initialize root user with password - 'passw0rd'  Expose port 22 of the VM (ssh) to port 2222 of eve for ssh:
+``` 
+eden pod deploy -p 2222:22 http://cdimage.debian.org/cdimage/openstack/current/debian-10.4.3-20200610-openstack-arm64.qcow2 -v debug --metadata='#cloud-config\npassword: passw0rd\nchpasswd: { expire: False }\nssh_pwauth: True\n'
```

Comment:
Please rewrite this one for Qemu variant (amd64 and 8027/tcp or 8028/tcp external port for example). Provided variant is for rpi (next PR)